### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by Githiba2013** [open-spaced-repetition/fsrs4anki#514]
My request is to further enhance the postponement feature and make it automatic, similar to SuperMemo's **auto-postpone**. 

The postponement feature is essential from a learning perspective. Postponing is necessary as it allows us to prioritize and allocate time for different tasks. On the other hand, our lives are not limited to incremental reading and reviewing flashcards.Sometimes we may need to set aside a few days to focus on other activities or even take a short break.

If a delay of a few days disrupts the entire memory algorithm, it would be counterproductive. Therefore, I hope that FSRS can incorporate an automatic postponement feature.

**Comment by Vilhelm-Ian** [comment]
What do you mean automatic postpone? If you missed a day to postpone all your cards?

**Comment by L-M-Sherlock** [comment]
> My request is to further enhance the postponement feature and make it automatic, similar to SuperMemo's **auto-postpone**.

Anki doesn't have priority system. So it's impossible to implement the auto-postpone of SuperMemo.

**Comment by Githiba2013** [comment]
> 推迟

Yes. Also, if I haven't finished reviewing all the cards, it will postpone the remaining cards automatically.

**Comment by Githiba2013** [comment]
> What do you mean automatic postpone? If you missed a day to postpone all your cards?
> 自动推迟是什么意思？如果您错过了推迟所有卡片的一天？

Yes. And if I haven't finished reviewing all the cards, it will postpone the remaining cards automatically.

**Comment by Vilhelm-Ian** [comment]
you can type in the browser this prop:due=-1 and select all and bury

**Comment by Expertium** [comment]
> Anki doesn't have priority system.

By the way, I suggested to implement a priority queue in FSRS some time ago. Maybe we should consider that.
